README.md

```diff
@@ -106,21 +117,21 @@ For more information on cases you should handle during layout, please refer to [
 
 FluentDarkModeKit will notify views or view controllers in the current window when the theme changes by calling the following delegate method. 
 
-Swift
 ```swift
 protocol DMTraitEnvironment: NSObjectProtocol {
     func dmTraitCollectionDidChange(_ previousTraitCollection: DMTraitCollection?)
 }
 ```
 
-Objective-C
-```objc
-@protocol DMTraitEnvironment <NSObject>
+## Migration
 
-- (void)dmTraitCollectionDidChange:(nullable DMTraitCollection *)previousTraitCollection;
+If you are using a pre 0.5.2 version and wants to migrate to the new version, here are the changes that you should pay attention to:

```

Comment:
Do you think we need to bump the major version to 1.0? Since most projects are configured to automatically update to the latest minor version, there would be many cases where developers update FluentDarkModeKit to this version with many breaking changes.

cc @icodesign @li-bei

Reply:
I agree we should bump a version. would be done in a separate PR.